src/utils/common.js

```diff
@@ -471,7 +470,8 @@ export const groupSectionsByDate = (
   const sections: { [string]: SectionData } = {};
 
   orderBy(data, [dateField], [sortDirection]).forEach((item) => {
-    const date = new Date(item[dateField] * timestampMultiplier);
+    const safeTimestamp = +parseDate(item[dateField]);
```

Comment:
Using unary `+` seems a bit hard to read. Would you conside using `parseDate(...).getTime()`? Alternatively could we add `getTimestamp(Date|string|number)`?

Reply:
With current implementation, safeTimestamp is a not adequate name, as the parsed date might be invalid and hence timestamp might be `NaN`.

Reply:
Replaced this with `parseTimestamp`.